backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/AccountRequestController.java

```diff
@@ -156,6 +163,19 @@ private String checkForDuplicateOrg(String organizationName, String state, Strin
     return String.join("-", organizationName, state);
   }
 
+  private List<String> getOrgAdminUserEmails(Organization org) {
+    List<String> adminUserEmails;
+    if (_ffc.isOktaMigrationEnabled()) {
+      adminUserEmails =
+          _das.getOrgAdminUsers(org).stream()
```

Comment:
nit: when looking over this code I had to scroll up further to see what `_das` was and I think the readability would feel more fluid renaming this to like `_dbAuthService` even though it's a longer variable name. Not that these all have to get changed right now but same feelings for `_ffc` to `_featureFlagConfig`, `_os` to `_orgService`, `_oqs` to `_orgQueueService`, `_aus` to `_apiUserService`, and `_es` to `_emailService`

Reply:
OOO good call out -- I can clean this up!

Reply:
Done!